### DISCUSSION TITLE
Promblem with PIL

### DISCUSSION THREAD

**Discussion by CreeperFarm** [microsoft/pylance-release#2534] [Q&A] [answered]
### Hi,
I have a problem, I can't run my python programme caused to that `from PIL import Image` who is needed inside my code to create image.
Can you help me plz.

**Comment by gramster** [comment]
It doesn't sound like this has to do with Pylance. Could you provide more details?

**Comment by bschnurr** [comment, marked as answer]
try `pip install pillow` in your virtual python environment

**Reply by CreeperFarm** [reply]
Thanks, now it's work
